presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraIntegrationSmokeTest.java

```diff
@@ -335,7 +335,7 @@ public void testKeyspaceNameAmbiguity()
         // There is no way to figure out what the exactly keyspace we want to retrieve tables from
         assertQueryFailsEventually(
                 "SHOW TABLES FROM cassandra.keyspace_3",
-                "More than one keyspace has been found for the case insensitive schema name: keyspace_3 -> \\(KeYsPaCe_3, kEySpAcE_3\\)",
+                ".*More than one keyspace has been found for the case insensitive schema name: keyspace_3 -> \\(KeYsPaCe_3, kEySpAcE_3\\)",
```

Comment:
are all the test regex changes still needed since you changed to propagate PrestoExceptions instead of wrapping them?  If so, what was the exception being thrown for these cases?  I want to make sure we're not changing exception types.

Reply:
oh, i think node location is helpful because it tells people where in the query the error is.  Is that possible to keep?

Reply:
discussed offline that supporting node location will be done as a follow up.

Reply:
We will have to preserve NodeLocation or TableNode while collecting table names in the MetadataExtractor. I will keep that as a follow up PR.